oauth2_lib/fastapi.py

```diff
@@ -209,13 +213,12 @@ async def authenticate(self, request: HTTPConnection, token: Optional[str] = Non
                 token_or_extracted_id_token = token
             else:
                 request = cast(Request, request)
+
                 if await self.is_bypassable_request(request):
                     return None
+
                 if token is None:
-                    extracted_id_token = await self.id_token_extractor.extract(request)
```

Comment:
How can a user set `auto_error=False`?

Reply:
I removed this functionality to simplify the code and its usage. This way, we don't have to deal with the confusing parameter, but we can still override `authenticate` and `extract` if needed.